doc/locale/en/api_reference.pot

```diff
@@ -139,7 +139,10 @@ msgstr ""
 msgid "Registers a new summary. Quantile computation is based on the algorithm `\"Effective computation of biased quantiles over data streams\" <https://ieeexplore.ieee.org/document/1410103>`_"
 msgstr ""
 
-msgid "Quantiles to observe in the form ``{quantile = error, ... }``. For example: ``{[0.5]=0.01, [0.9]=0.01, [0.99]=0.01}``"
+msgid "A list of 'targeted' :math:`\\\\varphi`-quantiles in the form ``{quantile = error, ... }``. For example: ``{[0.5]=0.01, [0.9]=0.01, [0.99]=0.01}``. A targeted :math:`\\\\varphi`-quantile is specified in the form of a :math:`\\\\varphi`-quantile and tolerated error. For example a ``{[0.5] = 0.1}`` means that the median (= 50th percentile) should be returned with 10 percent error. Note that percentiles and quantiles are the same concept, except percentiles are expressed as percentages. The :math:`\\\\varphi`-quantile must be in the interval [0, 1]. Note that a lower tolerated error for a :math:`\\\\varphi`-quantile results in higher usage of resources (memory and cpu) to calculate the summary."
+msgstr ""
+
+msgid "Table of summary parameters, used for configuring sliding window of time. 'Sliding window' consists of several buckets to store observations. New observation are added to each bucket. After a time period, 'head' bucket (bucket from which observations are collected) is reset and next bucket becomes new 'head'. I.e. each bucket will be store observations for the last ``max_age_time * age_buckets_count`` seconds before it will be reset. ``max_age_time`` set the duration of each bucket lifetime, i.e., how long observations are kept before they are discarded, in seconds ``age_buckets_count`` set the number of buckets of the time window. It determines the number of buckets used to exclude observations that are older than ``max_age_time`` from the Summary. The value is a trade-off between resources (memory and cpu for maintaining the bucket) and how smooth the time window is moved. Default value is `{max_age_time = math.huge, age_buckets_count = 1}`"
```

Comment:
```suggestion
msgid "Table of summary parameters, used for configuring sliding window of time. 'Sliding window' consists of several buckets to store observations. New observation are added to each bucket. After a time period, the 'head' bucket (bucket from which observations are collected) is reset and the next bucket becomes a new 'head'. I.e. each bucket will be store observations for the last ``max_age_time * age_buckets_count`` seconds before it will be reset. ``max_age_time`` set the duration of each bucket lifetime, i.e., how long observations are kept before they are discarded, in seconds ``age_buckets_count`` set the number of buckets of the time window. It determines the number of buckets used to exclude observations that are older than ``max_age_time`` from the Summary. The value is a trade-off between resources (memory and CPU for maintaining the bucket) and how smooth the time window is moved. Default value is `{max_age_time = math.huge, age_buckets_count = 1}`"
```

Reply:
It's autogenerated doc, I fixed it in original file